src/js/angular/rest/ttyg.rest.service.js

```diff
@@ -8,6 +8,7 @@ TTYGRestService.$inject = ['$http'];
 
 const CONVERSATIONS_ENDPOINT = 'rest/chat/conversations';
 const AGENTS_ENDPOINT = 'rest/chat/agents';
+const EXPLAIN_RESPONSE_ENDPOINT = 'rest/chat/conversations/explain';
```

Comment:
it's minor but you can reuse the CONVERSATIONS_ENDPOINT and concat the last segment

Reply:
Done